File: Java/src/main/java/Solution301_400/Solution344.java
package Solution301_400;

/**
 * @author windows
 */
public class Solution344 {
    public void reverseString(char[] s) {
        for (int i = 0; i < s.length / 2; i++) {
            char temp = s[i];
            s[i] = s[s.length - 1 - i];
            s[s.length - 1 - i] = temp;
        }
    }


    public static void main(String[] args) {
        Solution344 solution344 = new Solution344();
    }
}

File: C++/src/Solution301_400/Solution344.cpp
//
// Created by windows on 2023/3/2.
//

#include "Solution344.h"
#include <functional>
#include <iostream>
#include <vector>
#include <unordered_set>
#include <unordered_map>
#include <string>
using namespace std;

void Solution344::reverseString(vector<char> &s) {
    for (int i = 0, j = s.size() - 1; i < j; ++i, --j) {
        char c = s[i];
        s[i] = s[j];
        s[j] = c;
    }
}
